### PR TITLE
Make chart options observable

### PR DESCRIPTION
**Pull request by cs3b**
rebased against master including contribution made by @renerte and @broldak

### CODE REVIEW

**Comment by backspace** [comment]
Thanks for carrying this along periodically; it would fix a bug in my application, I hope it can be merged!

**Comment by Techn1x** [comment]
This would be great, would love to not need to rely on observers to update our xaxis categories

**Comment by schorke** [comment]
Can this essential feature be fixed & merged anytime soon ? Would love to get rid of observers !

cc @cs3b

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,6 +16,8 @@
 - Use the "app" argument if available
 - Bump bootstrap version to fix security vulnerability
 
+=======
```
Comment (nlfurniss) [inline CHANGELOG.md:19]: Please remove

File: package.json [hunk 2]
```diff
@@ -65,6 +65,9 @@
     "node": ">= 10.*"
   },
   "ember-addon": {
-    "configPath": "tests/dummy/config"
+    "configPath": "tests/dummy/config",
+    "versionCompatibility": {
+      "ember": ">=2.12.0"
```
Comment (nlfurniss) [inline package.json:70]: Please update to >=3.0.0

**Comment by broldak** [comment]
Would it be possible to merge this? @nlfurniss

**Comment by masterchef** [comment]
Please merge this in. Not having a chart redraw after options change is really annoying.

**Comment by nlfurniss** [comment]
Will be handled in #195

**Pull request closed without merging**